docs/uml/Style.puml

```diff
@@ -0,0 +1,5 @@
+!define LOGIC_COLOR #3333C4
+!define LOGIC_COLOR_T1 #7777DB
+!define LOGIC_COLOR_T2 #5252CE
+!define LOGIC_COLOR_T3 #1616B0
+!define LOGIC_COLOR_T4 #101086
```

Comment:
Maybe consider adding `hide footbox` to the `Style.puml` file so that the bottom boxes are hidden in sequence diagrams to reduce clutter?